### PR TITLE
For Python Upgrade - Update setup.py

### PR DESCRIPTION
**Pull request by jhanm12**
For Python 3.11 Upgrade

### CODE REVIEW

**Review by nishantupadhyay123 — APPROVED** [review]
please change the docs too.

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from yangkit.__version__ import __version__
 
 INSTALL_REQUIREMENTS = [
-    'lxml==3.4.4',
+    'lxml==4.9.3',
```
Comment (akshansh237) [inline setup.py:6]: i think we should remove the strict version check here

Comment (jhanm12) [reply setup.py:6]: Shall we proceed with this - https://github.com/yang-infra/yangkit/pull/12? (Remove this strict version check for all the packages or add '>=' version condition with the older versions?)

Comment (nishantupadhyay123) [reply setup.py:6]: this is cool when nobody is consuming the yangkit in their frameworks or code.
Just as an example , look at onetopo ..we have to put a condition for 3.6.0 for pyVXR as packages were not compatible.
I have always hesitated  unless you have a full blow UT.

Comment (akshansh237) [reply setup.py:6]: ok. i think we should try to put >= instead of just the module name.

**Comment by akshansh237** [comment]
pr shows old commits as diffs of this branch. can you fix your branch ?

**Pull request closed without merging**